openshift-ci/build-root/console/Dockerfile

```
@@ -1,7 +1,7 @@
-FROM quay.io/coreos/tectonic-console-builder:v25 AS build
+FROM quay.io/coreos/tectonic-console-builder:v29 AS build
 
 RUN mkdir -p /go/src/github.com/openshift/
-RUN cd /go/src/github.com/openshift && git clone https://github.com/openshift/console.git
+RUN cd /go/src/github.com/openshift && git clone https://github.com/openshift/console.git && cd console && git checkout release-4.16
```

Nit: the following changes simplify the same action:
```suggestion
RUN cd /go/src/github.com/openshift && git clone https://github.com/openshift/console.git -b release-4.16
```